authorizer/backup.go

```diff
@@ -23,26 +23,24 @@ func NewBackupService(s influxdb.BackupService) *BackupService {
 	}
 }
 
-func (b BackupService) CreateBackup(ctx context.Context) (int, []string, error) {
+func (b BackupService) BackupKVStore(ctx context.Context, w io.Writer) error {
 	span, ctx := tracing.StartSpanFromContext(ctx)
 	defer span.Finish()
 
+	// TODO(bbj): Correct permissions.
 	if err := IsAllowedAll(ctx, influxdb.ReadAllPermissions()); err != nil {
-		return 0, nil, err
+		return err
 	}
-	return b.s.CreateBackup(ctx)
+	return b.s.BackupKVStore(ctx, w)
 }
```

Comment:
@stuartcarnie I don't know the permissions we're using too well so I just left this as a `TODO`. Should this be `OwnerPermissions()`? Is there a better permission set for backup/restore?